src/components/Header.tsx

```diff
@@ -10,7 +11,7 @@ const Header: React.FC = () => {
   const navigation = [
     { name: 'Home', href: '/', icon: Home },
     { name: 'Leaderboard', href: '/leaderboard', icon: Trophy },
-    { name: 'GitHub', href: 'https://github.com/RouteWorks', icon: Github, isExternal: true },
+    { name: 'GitHub', href: contactInfo.github, icon: Github, isExternal: true },
```

Comment:
I guess here is just temporary. We will update it to the public repo after we release.